base-infrastructure/.github/workflows/destroy.yml

```diff
@@ -43,7 +43,7 @@ jobs:
           make tf-destroy env=dev \
           bucket=terraform-state-dev-${{ env.AWS_REGION }}-${{ env.AWS_DEV_ACCOUNT_NUMBER }} \
           region=${{ env.AWS_REGION }} \
-          key=${{ env.S3_PREFIX }}-dev.state \
+          key=mlops-terraform-dev.state \
```

Comment:
@jmgrubb it seems we made a mistake here when making the s3 bucket name configurable? This env var should be used for s3 bucket name instead of key. 

For now, if you don't want to merge this change and implement a fix, give me a shout.